mava/utils/sort_utils.py

```diff
@@ -27,6 +27,7 @@ def sample_new_agent_keys(
     agents: List,
     network_sampling_setup: List,
     net_keys_to_ids: Dict[str, int] = None,
+    fix_sampler: Optional[List] = None,
```

Comment:
@DriesSmit Can we document what this does ?

Reply:
Added it.